lib/ansible/cli/galaxy.py

```diff
@@ -498,6 +498,10 @@ def add_install_options(self, parser, parents=None):
             install_parser.add_argument('--ignore-signature-status-code', dest='ignore_gpg_errors', type=str, action='append',
                                         help=ignore_gpg_status_help, default=C.GALAXY_IGNORE_INVALID_SIGNATURE_STATUS_CODES,
                                         choices=list(GPG_ERROR_MAP.keys()))
+            install_parser.add_argument('--offline', dest='offline', action='store_true', default=False,
+                                        help='Install collection artifacts (tarballs) without contacting any distribution servers. '
+                                             'This does not apply to collections in Git repositories or prevent fetching remote URLs. '
```

Comment:
> fetching remote URLs

Could you clarify what's meant by this? Does it refer to _Git_ remote URLs? My brain is a bit stuck trying to parse this. Usually, when such confusions happen, splitting a sentence into two helps. WDYT?

Oh, and there's an accidental trailing space added to this string, I think...

```suggestion
                                             'This does not apply to collections in Git repositories or prevent fetching them from remote Git URLs.'
```

Does this catch the meaning correctly?

Reply:
It has no effect on URLs to remote tarballs or remote git URLs, 2 separate things.

Does this read better to you?
```suggestion
                                             'This does not apply to collections in remote Git repositories or URLs to remote tarballs.'
```

Reply:
Yes, much better. It didn't even occur to me that the original phrase was talking about the "tarball URLs". Now it makes total sense.